docs/usage.rst

```diff
@@ -94,22 +100,27 @@ You can also customize the template directory from a test by passing in the opti
 
    @pytest.fixture
    def custom_template(tmp_path) -> Path:
```

Comment:
I found some typos in the current `custom_template` implementation (a missing `}` character, `tmp` used instead of `tmp_path` which made an undefined variable error, etc.).

I've also made some changes for clarity. I found the difference between "tutu"/"toto" difficult to catch while skim reading, so I've renamed one "toto" -> "default_name" instead.

Reply:
toto/tutu is very common french speaker dummy names (it's actualy titi, tutu, toto, tata). Sorry if it translates poorly into english.

Reply:
> toto/tutu is very common french speaker dummy names (it's actualy titi, tutu, toto, tata)

Oh, that's cute! Today I learned a new thing 😄 
I can change it back to be more french sounding. We could use capitalization to make the difference a bit more immediately obvious ("TOTO"/"tutu")